test/Api.Tests/TestStartup.cs

```diff
@@ -1,20 +1,69 @@
 ﻿using Api.Data;
+using Api.Utility;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System.IO;
 
 namespace Api.Tests
 {
-    public class TestStartup : Startup
+    public class TestStartup
     {
-        public TestStartup(IHostingEnvironment env) : base(env) { }
+        public TestStartup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)                
+                .AddEnvironmentVariables();
+
+            Configuration = builder.Build();
+        }
+
+        public IConfigurationRoot Configuration { get; }
 
-        public void ConfigureTestServices(IServiceCollection services)
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
         {
-            base.ConfigureServices(services);
+            // Read configurations from json
+            var appConfig = new AppConfig(Configuration);
```

Comment:
You should consider, which fields do you need from the appsettings.json/enviorment here. Maybe it it better to create an own TestConfig class and related a testconfig.json file.